### ISSUE TITLE
Items.Add is not work correctly of ComboBoxEx

### ISSUE THREAD

**Issue by chunmingwang** [XusinboyBekchanov/MyFbFramework#16]
I found Items.Add is not work correctly in Windows of ComboBoxEx

**Comment by XusinboyBekchanov** [comment]
How does it not work?

**Comment by chunmingwang** [comment]
I don't have a solution yet

**Comment by XusinboyBekchanov** [comment]
I tried, a new item is added.

**Comment by XusinboyBekchanov** [comment]
Can you send your example? I am trying to compile on my own.

**Comment by chunmingwang** [comment]
I had send you a test example for ComboBoxEx to your email bxusinboy@mail.ru, please check your mail box.

**Comment by chunmingwang** [comment]
Hi Xusinboy,


The attached is my test example, FYI.
Please check it.




--

Best regards,

Cm Wang (王春明)

诚以修身 信以立世




At 2021-12-21 13:52:54, "Xusinboy Bekchanov" ***@***.***> wrote:

Can you send your example? I am trying to compile on my own.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by XusinboyBekchanov** [comment]
Fixed: ComboBoxEx List Height:
https://github.com/XusinboyBekchanov/MyFbFramework/commit/44800d4e4eaa83f45f413f1835974c51e3c3d77b

Prior to this fix, I just set the height of the ComboBoxEx higher than 22.

**Comment by chunmingwang** [comment]
I tested it in different Style, the result are:
cbSimple - not work 
cbDropDown - not work 
cbDropDownList - fixed
cbOwnerDrawFixed - fixed
cbOwnerDrawVariable - not work

**Comment by chunmingwang** [comment]
Hi Xusinboy,


for ComboBoxEx(style: cbSimple) 
I found some issue see as following.
1st open Form in desing view, display normal.






2nd click and active ComboBoxEx(style: cbSimple) 
then display become abnormal.

--

Best regards,

Cm Wang (王春明)

诚以修身 信以立世




At 2021-12-21 18:57:08, "Xusinboy Bekchanov" ***@***.***> wrote:

Fixed: ComboBoxEx List Height:
44800d4

Prior to this fix, I just set the height of the ComboBoxEx higher than 22.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
